cfgov/legacy/views/complaint.py

```diff
@@ -24,3 +24,17 @@ def get_context_data(self, **kwargs):
         })
 
         return context
+
```

Comment:
And this would add the landing page to the cache-tag party:
```suggestion

    def dispatch(self, *args, **kwargs):
        response = super(ComplaintLandingView, self).dispatch(*args, **kwargs)
        response['Edge-Cache-Tag'] = 'complaints'
        return response

```